scalyr_agent/agent_main.py

```diff
@@ -907,19 +908,23 @@ def __run(self, controller):
                     logs_initial_positions = None
 
                 # 2->TODO it was very helpful to see what python version does agent run on. Maybe we can keep it?
-
                 python_version_str = sys.version.replace("\n", "")
-
-                log.info(
-                    "Starting scalyr agent... (version=%s) %s (Python version: %s)"
-                    % (SCALYR_VERSION, scalyr_util.get_pid_tid(), python_version_str)
-                )
-                log.log(
-                    scalyr_logging.DEBUG_LEVEL_1,
-                    "Starting scalyr agent... (version=%s) %s (Python version: %s)"
-                    % (SCALYR_VERSION, scalyr_util.get_pid_tid(), python_version_str),
+                build_revision = get_build_revision()
+
+                # TODO: Why do we log the same line under info and debug? Intentional?
```

Comment:
Maybe it is because all debug output goes to `agent_debug.log` and we thought that normal non-debug logging wouldn't get there.. so to be sure that we had the version number, etc in `agent_debug.log`, we added this in?